src/Rhetos.CommonConcepts/Dom.DefaultConcepts/DataStructureReadParameters.cs

```diff
@@ -136,7 +136,7 @@ private static void AddSimplifiedTypeNames(ICollection<DataStructureReadParamete
                 foreach (var namePart in _namePartsRegex.Matches(shortName).Reverse()) // Reverse to avoid corrupting remaining matches after removing some prefixes.
                     shortName = TryRemovePrefix(shortName, namePart.Index, namePart.Length, removablePrefixes);
 
-                if (shortName.StartsWith("IEnumerable<", StringComparison.Ordinal) && shortName.EndsWith(">", StringComparison.Ordinal))
+                if (shortName.StartsWith("IEnumerable<", StringComparison.Ordinal) && shortName.EndsWith('>'))

```

Comment:
https://chat.openai.com/share/878eea0f-7dce-401f-8aad-c2a5eb214f0a

GPT says that this is safe